modules/04_PE_FE_parameters/iea2014/datainput.gms

```diff
@@ -24,11 +24,109 @@ $offdelim
+                                                            /  p04_IO_output_beforeFix_Total("2005",regi,"fesoi");
+
+
+f04_IO_output("2005",regi,"sesofos","fesoi","tdfossoi")$(p04_IO_output_beforeFix_Total("2005",regi,"fesoi")) = p04_IO_output_beforeFix("2005",regi,"sesofos","fesoi","tdfossoi")  
+                                                            *  ( pm_fedemand("2005",regi,"feso_otherInd")
+                                                               + pm_fedemand("2005",regi,"feso_cement")
+                                                               + pm_fedemand("2005",regi,"feso_steel")
+                                                               + pm_fedemand("2005",regi,"feso_chemicals"))
+                                                            /  p04_IO_output_beforeFix_Total("2005",regi,"fesoi");
+
+*** industry liquids
+p04_IO_output_beforeFix_Total(t,regi,"fehoi") = p04_IO_output_beforeFix(t,regi,"seliqbio","fehoi","tdbiohoi")
+                                                  + p04_IO_output_beforeFix(t,regi,"seliqfos","fehoi","tdfoshoi");
+
+
+f04_IO_output("2005",regi,"seliqbio","fehoi","tdbiohoi")$(p04_IO_output_beforeFix_Total("2005",regi,"fehoi")) = p04_IO_output_beforeFix("2005",regi,"seliqbio","fehoi","tdbiohoi")  
+                                                            *  ( pm_fedemand("2005",regi,"feli_otherInd")
+                                                               + pm_fedemand("2005",regi,"feli_cement")
+                                                               + pm_fedemand("2005",regi,"feli_steel")
+                                                               + pm_fedemand("2005",regi,"feli_chemicals"))
+                                                            /  p04_IO_output_beforeFix_Total("2005",regi,"fehoi");
+
+
+f04_IO_output("2005",regi,"seliqfos","fehoi","tdfoshoi")$(p04_IO_output_beforeFix_Total("2005",regi,"fehoi")) = p04_IO_output_beforeFix("2005",regi,"seliqfos","fehoi","tdfoshoi")  
+                                                            *  ( pm_fedemand("2005",regi,"feli_otherInd")
+                                                               + pm_fedemand("2005",regi,"feli_cement")
+                                                               + pm_fedemand("2005",regi,"feli_steel")
+                                                               + pm_fedemand("2005",regi,"feli_chemicals"))
+                                                            /  p04_IO_output_beforeFix_Total("2005",regi,"fehoi");
+
+*** industry gases
+p04_IO_output_beforeFix_Total(t,regi,"fegai") = p04_IO_output_beforeFix(t,regi,"segabio","fegai","tdbiogai")
+                                                  + p04_IO_output_beforeFix(t,regi,"segafos","fegai","tdfosgai");
+
+
+f04_IO_output("2005",regi,"segabio","fegai","tdbiogai")$(p04_IO_output_beforeFix_Total("2005",regi,"fegai")) = p04_IO_output_beforeFix("2005",regi,"segabio","fegai","tdbiogai")  
```

Comment:
I do not like that this is a change of pure input data (f*). Could we use a parameter for this?

Reply:
The original data are still in ``p04_IO_output_beforeFix``. Can be aligned once temporary solution not needed anymore.